<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessWidget</class>
 <widget class="QWidget" name="ProcessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bottomState">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>645</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>32</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="processInfo">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>2</y>
     <width>108</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>搜索进程名/PID..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTerminateProcess">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>2</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>结束进程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFresh">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>on_btnTerminateProcess_clicked()</slot>
 </slots>
</ui>
